src/functions/lookupRandomCocktail.js

```diff
@@ -0,0 +1,25 @@
+const dotenv = require("dotenv");
+const axios = require("axios");
+
+dotenv.config();
+
+function lookupRandomCocktail (){
+    const options = {
+        "port": null,
+        "headers": {
+            "X-RapidAPI-Key": process.env.X_RAPIDAPI_KEY,
+            "X-RapidAPI-Host": "the-cocktail-db.p.rapidapi.com",
+            "useQueryString": true
+        }
+    };
```

Comment:
options seems constant, you can take it out from the function

Reply:
I modified the code and created another pull request #40
Could you check the code modification in the new pull request?